lib/common/tests/xml/pcmk__xe_set_id_test.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2024 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU General Public License version 2
+ * or later (GPLv2+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+
+#include <crm/common/unittest_internal.h>
+#include <crm/common/xml_internal.h>
+
+static void
+null_node(void **state)
+{
+    pcmk__xe_set_id(NULL, "test_id");
```

Comment:
I'm surprised cmocka lets you get by without an assertion. Could return a standard code just for the sake of unit testing, but not really worth it